packages/passport-interface/src/RequestTypes.ts

```diff
@@ -75,6 +94,30 @@ export interface DownloadEncryptedStorageRequest {
    * the encryption key.
    */
   blobKey: string;
+
+  /**
+   * Optional field indicating the revision of the latest blob already known to
+   * the client.  If this matches the latest blob stored on the server, the
+   * request will succeed, but the result will not contain any blob.
+   */
+  knownRevision?: string;
```

Comment:
it feels like `baseRevision` and `knownRevision` refer to the same concept and maybe should be named the same thing, but I can also see why you did it this way too.

Reply:
They're not quite the same concept in the context they're used, but they're both "the most recent revision this client downloaded from the server" so maybe I need to think of a good name for that.